src/Uno.UI/UI/Xaml/Controls/TreeView/TreeViewList.cs

```diff
@@ -0,0 +1,689 @@
+						itemAfterInsertPositionString = GetAutomationName(afterInsertIndex);
+					}
+
+					m_dropTargetDropEffectString = BuildEffectString(itemBeforeInsertPositionString, itemAfterInsertPositionString, dragItemString, draggedOverString);
+				}
+
+				if (!forceUpdate && oldValue != m_dropTargetDropEffectString)
+				{
+					AutomationPeer treePeer = FrameworkElementAutomationPeer.FromElement(this);
+					treePeer.RaisePropertyChangedEvent(DropTargetPatternIdentifiers.DropTargetEffectProperty, oldValue, m_dropTargetDropEffectString);
+				}
+			}
+		}
+
+		internal void EnableMultiselect(bool isEnabled)
+		{
+			m_isMultiselectEnabled = isEnabled;
+		}
+
+		internal bool IsMultiselect => m_isMultiselectEnabled;
+
+		private bool IsMutiSelectWithSelectedItems()
+		{
+			//    var selectedItems = ListViewModel().GetSelectedNodes();
+			//bool isMutiSelect = m_isMultiselectEnabled && selectedItems.Size() > 0;
+			//    return isMutiSelect;
+			throw new NotImplementedException();
+		}
+
+		private bool IsSelected(TreeViewNode node)
+		{
+			bool isSelected = false;
+			var selectedItems = ListViewModel.SelectedNodes;
+			for (var i = 0; i < selectedItems.Count; i++)
+			{
+				if (selectedItems[i] == node)
+				{
+					isSelected = true;
+					break;
+				}
+			}
+
+			return isSelected;
+		}
+
+		internal IList<TreeViewNode> GetRootsOfSelectedSubtrees()
+		{
+			//    std.vector<TreeViewNode> roots;
+			//var selectedItems = ListViewModel().GetSelectedNodes();
+			//    for (unsigned int i = 0; i<selectedItems.Size(); i++)
+			//    {
+			//        var item = selectedItems.GetAt(i);
+			//var selectionRoot = GetRootOfSelection(item);
+			//var it = std.find(roots.cbegin(), roots.cend(), selectionRoot);
+			//        if (it == roots.cend())
+			//        {
+			//            roots.emplace_back(selectionRoot);
+			//        }
+			//    }
+
+			//    return roots;
+			throw new NotImplementedException();
+		}
+
+		private int FlatIndex(TreeViewNode node)
+		{
+			var flatIndex = 0;
+			bool found = ListViewModel.IndexOfNode(node, flatIndex);
+			flatIndex = found ? flatIndex : -1;
+			return flatIndex;
+		}
+
+		private bool IsFlatIndexValid(int index)
+		{
+			//    return index >= 0 && index<static_cast<int>(ListViewModel().Size());
+			throw new NotImplementedException();
+		}
+
+		private int RemoveNodeFromParent(TreeViewNode node)
+		{
+			var children = node.Parent.Children;
+			int indexInParent = children.IndexOf(node);
+
+			if (indexInParent > -1)
+			{
+				children.RemoveAt(indexInParent);
+			}
+
+			return indexInParent;
+		}
+
+		private bool IsIndexValid(int index)
+		{
+			//	return index >= 0 && index < (int)Items().Size();
+			throw new NotImplementedException();
+		}
+
+		private string GetAutomationName(int index)
+		{
+			TreeViewItem item = null;
+			string automationName = "";
+
+			if (IsIndexValid(index))
+			{
+				var interim = ContainerFromIndex(index) as TreeViewItem;
+				if (item != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'false'; some subsequent code is never executed.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)